src/components/about-us/index.js

```diff
@@ -1,13 +1,32 @@
 import Footer from '@twreporter/react-components/lib/footer'
 import React, { PureComponent } from 'react'
 import Section02 from './section-02'
+import Section03 from './section-03'
 import WebFont from './web-font'
+import { injectGlobal } from 'styled-components'
 
 export class AboutUs extends PureComponent {
+  componentWillMount() {
+    // to prevent horizontal overscrolling
+    injectGlobal`
+      html, body {
+        overflow-x: hidden;
+      }
+    `
+  }
+  componentWillUnmount() {
+    injectGlobal`
+      html, body {
+        overflow-x: visible;
+      }
+    `
+  }
```

Comment:
I found that adding a container to `about-us` with `width: 100%` and `overflow-x: hidden` has the same effect to prevent overscroll. So there is no need to use `injectGlobal` that break the pattern.